packages/modular-scripts/src/build/index.ts

```diff
@@ -135,35 +133,25 @@ async function buildStandalone(
     // create-react-app doesn't support plain module outputs yet,
     // so --preserve-modules has no effect here
 
-    const buildScript = require.resolve(
-      'modular-scripts/react-scripts/scripts/build.js',
-    );
+    // If it's an app, set it at ESBUILD_TARGET_FACTORY or default to es2015
+    // If it's not an app it's an ESM view, then we need es2020
+    const esbuildTargetFactory = isApp ? browserTarget : ['es2020'];
```

Comment:
Casual observation:
Webpack build logic uses 'isApp' a lot, ie if not app, then build as if it was an ESM view etc
This would potentially cause problems/not be ideal if adding more types - I feel like we should be doing every time things like if (type === app) else if (type === esmView) else Error Type not supported or similar, and handle every type we have so far - 
unless we decide its absolutely impossible another type will ever go through the build process